vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -90,6 +90,9 @@ type VerticalPodAutoscalerSpec struct {
 	// resources for all containers in the pod, without additional constraints.
 	// +optional
 	ResourcePolicy *PodResourcePolicy `json:"resourcePolicy,omitempty" protobuf:"bytes,3,opt,name=resourcePolicy"`
+
+	// Denote the recommender to be used
+	RecommenderName []string `json:"recommenderName,omitempty" protobuf:"bytes,1,opt,name=recommenderName"`
```

Comment:
1 is already used for TargetRef, it should 4 as in KEP.

Reply:
Yes, it should be 4. Thanks.